packages/vm/src/runBlock.ts

```diff
@@ -554,7 +554,7 @@ export async function accumulateParentBlockHash(
       'Cannot call `accumulateParentBlockHash`: EIP 2935 is not active',
     )
   }
-  const historyAddress = new Address(bigIntToAddressBytes(vm.common.param('historyStorageAddress')))
+  const historyAddress = new Address(bigIntToAddressBytes(vm.common.param('systemAddress')))
```

Comment:
why do we need this change?

Reply:
We probably don't.  I think it got carried over from previous PRs related to EIP-.2935